package.json

```diff
@@ -114,15 +115,15 @@
 		"@hashicorp/react-alert-banner": "^7.0.5",
 		"@hashicorp/react-button": "^6.3.1",
 		"@hashicorp/react-code-block": "^6.3.0",
-		"@hashicorp/react-consent-manager": "^9.0.1",
-		"@hashicorp/react-docs-page": "^17.8.0",
+		"@hashicorp/react-consent-manager": "^9.1.1",
+		"@hashicorp/react-docs-page": "^17.9.0",
 		"@hashicorp/react-docs-sidenav": "^9.1.0",
 		"@hashicorp/react-enterprise-alert": "^7.0.1",
 		"@hashicorp/react-error-view": "^0.2.0",
 		"@hashicorp/react-hashi-stack-menu": "^3.0.0",
 		"@hashicorp/react-head": "^3.3.2",
 		"@hashicorp/react-inline-svg": "^6.1.0",
-		"@hashicorp/react-marketo-form": "^0.8.2",
+		"@hashicorp/react-marketo-form": "^0.8.4",
```

Comment:
I bumped this to address a type error based on updated types for `window.analytics`